Geometry/HGCalCommonData/plugins/DDHGCalTBModuleX.cc

```diff
@@ -0,0 +1,364 @@
+#include <cmath>
+#include <algorithm>
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DetectorDescription/Core/interface/DDutils.h"
+#include "DetectorDescription/Core/interface/DDLogicalPart.h"
+#include "DetectorDescription/Core/interface/DDSolid.h"
+#include "DetectorDescription/Core/interface/DDMaterial.h"
+#include "DetectorDescription/Core/interface/DDCurrentNamespace.h"
+#include "DetectorDescription/Core/interface/DDSplit.h"
+#include "Geometry/HGCalCommonData/plugins/DDHGCalTBModuleX.h"
+#include "CLHEP/Units/GlobalPhysicalConstants.h"
+#include "CLHEP/Units/GlobalSystemOfUnits.h"
+
+//#define EDM_ML_DEBUG
+
+DDHGCalTBModuleX::DDHGCalTBModuleX() : factor_(0.5*sqrt(2.0)),
+				       tan30deg_(tan(30.0*CLHEP::deg)) {
```

Comment:
Other pull requests are replacing CLHEP. Why not this one?
Also, `0.5*sqrt(2.0)` should probably be a constant:
`constexpr double sqr2over2 = 0.5*sqrt(2.0);`

Reply:
Done